depends/packages/qt.mk

```diff
@@ -40,6 +40,7 @@ $(package)_config_opts += -bindir $(build_prefix)/bin
 $(package)_config_opts += -c++std c++17
 $(package)_config_opts += -confirm-license
 $(package)_config_opts += -hostprefix $(build_prefix)
+$(package)_config_opts += -ltcg
```

Comment:
I think this could be a good starting point, assuming our configure checks also work. I haven't tested yet.
```diff
--- a/depends/packages/qt.mk
+++ b/depends/packages/qt.mk
@@ -159,6 +159,9 @@ $(package)_config_opts_x86_64_linux = -xplatform linux-clang-libc++
 else
 $(package)_config_opts_x86_64_linux = -xplatform linux-g++-64
 endif
+ifneq ($(LTO),)
+$(package)_config_opts_x86_64_linux += -ltcg
+endif
 $(package)_config_opts_aarch64_linux = -xplatform linux-aarch64-gnu-g++
 $(package)_config_opts_powerpc64_linux = -platform linux-g++ -xplatform bitcoin-linux-g++
 $(package)_config_opts_powerpc64le_linux = -platform linux-g++ -xplatform bitcoin-linux-g++
```